modules/visual_script/visual_script_nodes.cpp

```diff
@@ -2455,7 +2455,7 @@ VisualScriptSceneNode::TypeGuess VisualScriptSceneNode::guess_output_type(TypeGu
 void VisualScriptSceneNode::_validate_property(PropertyInfo &property) const {
 #ifdef TOOLS_ENABLED
 	if (property.name == "node_path") {
-		Ref<Script> script = get_visual_script();
+		Ref<Script> script = get_container();
```

Comment:
Resource to Script casts happening implicitly. Considering it's used as Script is there a reason to use Resource type?

Reply:
I don't think that it makes a difference, there is a is valid check just below it.... :/

If there is a place without is_valid check then yes that will be a problem.